src/pages/tea-error-screen.html

```diff
@@ -0,0 +1,313 @@
+        }
+
+        .content-container > * {
+          max-width: 90%;
+        }
+      }
+
+      /* sm — 600 */
+      @media screen and (min-width: 600px) {
+        main {
+          padding: 32px;
+        }
+
+        .content-container {
+          flex-wrap: wrap;
+        }
+
+        .content-container > * {
+          max-width: 75%;
+        }
+      }
+
+      @media screen and (max-width: 899px) {
+        main {
+          padding: 32px;
+        }
+
+        .content-container {
+          flex-wrap: wrap;
+        }
+
+        .image-container {
+          align-self: flex-end;
+          position: static;
+        }
+      }
+
+      /* md — 900 */
+      @media screen and (min-width: 900px) {
+        main {
+          padding: 32px;
+        }
+
+        .content-container {
+          flex-wrap: nowrap;
+        }
+
+        .content-container > * {
+          max-width: 50%;
+        }
+      }
+
+      /* lg — 1200px */
+      @media screen and (min-width: 1200px) {
+        main {
+          padding: 32px;
+        }
+
+        .content-container {
+          max-width: 1000px;
+        }
+
+        .content-container > * {
+          max-width: 45%;
+        }
+
+        .text-container > * {
+          /* Help with readability by setting between 60-75ch */
+          /* https://piccalil.li/blog/line-length/ */
+          max-width: 54ch;
+        }
+
+        .text-container > h1 {
+          max-width: 35ch;
+        }
+      }
+
+      /* xl — 1536px */
+      @media screen and (min-width: 1536px) {
+        .content-container {
+          max-width: 1200px;
+        }
+
+        .content-container > * {
+          max-width: 40%;
+        }
+
+        .text-container > * {
+          /* Help with readability by setting between 60-75ch */
+          /* https://piccalil.li/blog/line-length/ */
+          max-width: 54ch;
+        }
+
+        .text-container > h1 {
+          max-width: 35ch;
+        }
+      }
+    </style>
+    <script src="./Maintenance_files/jquery-1.11.3.min.js"></script>
+    <script src="./Maintenance_files/moment.min.js"></script>
+    <script type="text/javascript">
+      $.get('https://bifrost.call-em-all.com/system_status', function (data) {
+        $('#status-message').html(
+          data.details ||
+            'We’re currently experiencing a service interruption, but our team is working hard to resolve it as soon as possible. Thank you for your patience and understanding!'
+        );
+
+        if (data.datetime)
+          $('#status-date').html(
+            moment(data.datetime).format('MMMM Do, YYYY h:mm a')
+          );
+        else $('#last-update').hide();
+      });
+    </script>
+  </head>
+
+  <body x-ms-format-detection="none">
+    <main>
+      <section class="content-container">
+        <div class="image-container">
+          <img
+            class="doodle-image"
+            src="https://cdn.text-em-all.com/assets/doodle-error-screen.png"
```

Comment:
Since these files will be hosted in S3, wouldn't it make sense to also host the images in S3?